lib/WP_Auth0_LoginManager.php

```diff
@@ -213,85 +213,42 @@ public function init_auth0() {
 	 *
 	 * @throws WP_Auth0_LoginFlowValidationException - OAuth login flow errors.
 	 * @throws WP_Auth0_BeforeLoginException - Errors encountered during the auth0_before_login action.
+	 * @throws WP_Auth0_InvalidIdTokenException If the ID token does not validate.
 	 *
 	 * @link https://auth0.com/docs/api-auth/tutorials/authorization-code-grant
-	 *
-	 * @throws WP_Auth0_InvalidIdTokenException If the ID token does not validate.
 	 */
 	public function redirect_login() {
-		$domain             = $this->a0_options->get( 'domain' );
```

Comment:
Lots of option gets and error handling moved into `WP_Auth0_Api_Exchange_Code` to standardize this call and provide much better logging.